SPECS/maven/maven.spec

```diff
@@ -133,6 +126,10 @@ cp %{_builddir}/apache-maven-%{version}/apache-maven/README.txt %{buildroot}%{_p
 %{_prefixmvn}/README.txt
 
 %changelog
+* Wed Feb 15 2023 Sumedh Sharma <sumsharma@microsoft.com> - 3.8.7-1
+- Update to version 3.8.7
+- Minor fixes in maven_build_caches script
```

Comment:
Please be more explicit about what the minor fixes were

Reply:
Removed.